sda-commons-server-dropwizard/src/test/java/org/sdase/commons/server/dropwizard/bundles/scanner/JacksonTypeScannerTest.java

```diff
@@ -0,0 +1,197 @@
+        .contains("AUTH_DISABLEAUTH (boolean)")
+        .contains("OPA_OPACLIENT_TIMEOUT (Duration)")
+        .contains("OPA_OPACLIENT_CONNECTIONTIMEOUT (Duration)")
+        .contains("OPA_OPACLIENT_CONNECTIONREQUESTTIMEOUT (Duration)")
+        .contains("OPA_OPACLIENT_TIMETOLIVE (Duration)")
+        .contains("OPA_OPACLIENT_COOKIESENABLED (boolean)")
+        .contains("OPA_OPACLIENT_NORMALIZEURIENABLED (boolean)")
+        .contains("OPA_OPACLIENT_MAXCONNECTIONS (int)")
+        .contains("OPA_OPACLIENT_MAXCONNECTIONSPERROUTE (int)")
+        .contains("OPA_OPACLIENT_KEEPALIVE (Duration)")
+        .contains("OPA_OPACLIENT_RETRIES (int)")
+        .contains("OPA_OPACLIENT_USERAGENT (String)")
+        .contains("OPA_OPACLIENT_VALIDATEAFTERINACTIVITYPERIOD (Duration)")
+        .contains("OPA_OPACLIENT_MINTHREADS (int)")
+        .contains("OPA_OPACLIENT_MAXTHREADS (int)")
+        .contains("OPA_OPACLIENT_WORKQUEUESIZE (int)")
+        .contains("OPA_OPACLIENT_GZIPENABLED (boolean)")
+        .contains("OPA_OPACLIENT_GZIPENABLEDFORREQUESTS (boolean)")
+        .contains("OPA_OPACLIENT_CHUNKEDENCODINGENABLED (boolean)")
+        .contains("OPA_OPACLIENT_PROXY_HOST (String)")
+        .contains("OPA_OPACLIENT_PROXY_SCHEME (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_USERNAME (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_PASSWORD (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_AUTHSCHEME (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_REALM (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_HOSTNAME (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_DOMAIN (String)")
+        .contains("OPA_OPACLIENT_PROXY_AUTH_CREDENTIALTYPE (String)")
+        .contains("OPA_OPACLIENT_TLS_PROTOCOL (String)")
+        .contains("OPA_OPACLIENT_TLS_PROVIDER (String)")
+        .contains("OPA_OPACLIENT_TLS_KEYSTOREPATH (File)")
+        .contains("OPA_OPACLIENT_TLS_KEYSTOREPASSWORD (String)")
+        .contains("OPA_OPACLIENT_TLS_KEYSTORETYPE (String)")
+        .contains("OPA_OPACLIENT_TLS_KEYSTOREPROVIDER (String)")
+        .contains("OPA_OPACLIENT_TLS_TRUSTSTOREPATH (File)")
+        .contains("OPA_OPACLIENT_TLS_TRUSTSTOREPASSWORD (String)")
+        .contains("OPA_OPACLIENT_TLS_TRUSTSTORETYPE (String)")
+        .contains("OPA_OPACLIENT_TLS_TRUSTSTOREPROVIDER (String)")
+        .contains("OPA_OPACLIENT_TLS_TRUSTSELFSIGNEDCERTIFICATES (boolean)")
+        .contains("OPA_OPACLIENT_TLS_VERIFYHOSTNAME (boolean)")
+        .contains("OPA_OPACLIENT_TLS_CERTALIAS (String)")
+        .contains("OPA_OPACLIENT_TLS_VALIDKEYSTOREPASSWORD (boolean)")
+        .contains("OPA_OPACLIENT_TLS_VALIDTRUSTSTOREPASSWORD (boolean)")
+        .contains("OPA_DISABLEOPA (boolean)")
+        .contains("OPA_BASEURL (String)")
+        .contains("OPA_POLICYPACKAGE (String)")
+        .contains("KAFKA_DISABLED (boolean)")
+        .contains("KAFKA_CONFIG_<KEY> (Map)")
+        .contains("KAFKA_PRODUCERS_<KEY>_CLIENTID (String)")
```

Comment:
We need some documentation/instruction for users. According to the examples in the other test, the key is case sensitive while the rest of the env variable needs to be upper cased.

Reply:
Clarified [here](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2685/commits/4ff51cfd8f9dfe112019b977544be1929db2c5fb).